### ISSUE TITLE
Automatic assignment of configuration issues

### ISSUE THREAD

**Issue by fgreinacher** [renovatebot/renovate#8359]
**What would you like Renovate to be able to do?**

I want to bring the right people's attention to errors in Renovate configuration files. 

**Did you already have any implementation ideas?**

It would be great if Renovate would assign the "Fix Renovate Configuration" issue to the person that has touched the Renovate configuration for the last time (based on Git history).

- [x] I'm willing to contribute this.

**Comment by rarkins** [comment]
Maybe and/or codeowner if one exists?

**Comment by fgreinacher** [comment]
> Maybe and/or codeowner if one exists?

Yes, that would also be an option. GitLab (and maybe other platforms?) can only handle a single assignee, so I think we should prefer the commit author over code owners.

Thinking a bit deeper I'm now wondering whether Renovate currently needs to map commit authors to platform users. If not this could be a challenge implementation-wise.

**Comment by rarkins** [comment]
> Thinking a bit deeper I'm now wondering whether Renovate currently needs to map commit authors to platform users ❓ If not this could be a challenge implementation-wise.

Yes, neither the commit author or committer are necessarily even members of the platform, although it's pretty rare if they wouldn't be. However you definitely can't be sure you can easily reverse engineer email to username regardless.
